.clang-format

```diff
@@ -0,0 +1,4 @@
+---
+BasedOnStyle: LLVM
```

Comment:
You're right, thinking about it a second time I hate their capitalization rules. Switching back to google.